### PR TITLE
Typescript-ify EmbedPreview

### PR DESCRIPTION
**Pull request by ghop02**

### CODE REVIEW

File: src/components/ShareBlock/EmbedPreview.tsx [hunk 1]
```diff
@@ -32,9 +42,9 @@ export default function EmbedPreview({ open, onClose, county }) {
   // When you click both copy links back to back, it'll trigger multiple snackbar
   // messages.  We show them serially using a queue per example at
   // https://material-ui.com/components/snackbars/#consecutive-snackbars
-  const snackbarQueueRef = useRef([]);
+  const snackbarQueueRef = useRef<Array<Message>>([]);
```
Comment (pnavarrc) [inline src/components/ShareBlock/EmbedPreview.tsx:45]: we tend to use `Message[]` in the code, but 🤷

File: src/components/ShareBlock/EmbedPreview.tsx [hunk 2]
```diff
@@ -79,12 +89,12 @@ export default function EmbedPreview({ open, onClose, county }) {
   return (
     <>
       <CenteredModal open={open} onClose={onClose}>
-        <EmbedPreviewStyled elevation="2">
+        <EmbedPreviewStyled>
```
Comment (pnavarrc) [inline src/components/ShareBlock/EmbedPreview.tsx:92]: why are we removing the elevation?

**Review by pnavarrc — APPROVED** [review]
Looks good!

File: src/components/ShareBlock/EmbedPreview.tsx (hunk 2 above)
Comment (ghop02) [reply src/components/ShareBlock/EmbedPreview.tsx:92]: it isn't used on the underlying styling

**Pull request merged**